scripts/generate-version-files.js

```diff
@@ -20,12 +22,19 @@ if (!generateOnly) {
   }
 
   // Do a dry-run on all packages
-  run('rush publish -a');
+  run(`${rushCmd} publish -a`);
   status = run('git status --porcelain=1');
-  modified = status
-    .split(/\n/g)
-    .map(line => line && '"' + line.trim().split(/\s/)[1] + '"')
-    .filter(item => item);
+  status.split(/\n/g).forEach(line => {
+    if (line) {
+      const parts = line.trim().split(/\s/);
+
+      if (parts[0] === '??') {
+        untracked.push(parts[1]);
```

Comment:
would be great to have comments on what kind of files would be untracked/modified

Reply:
adding a comment